src/DFGNode.cpp

```diff
@@ -127,7 +156,8 @@ StringRef DFGNode::getStringRef() {
 }
 
 bool DFGNode::isCall() {
-  if (m_opcodeName.compare("call") == 0 && !isVectorized())
+  // only fp2fx int2fx can be regarded as call. other calls like lut/quantize should be treated as special functionalities.
+  if (m_opcodeName.compare("call") == 0 && !isVectorized() && !isLut() && !isQuantize())
```

Comment:
Can you help remind me why `isVectorized()` is skipped here? I thought vectorized is recognized via `v` in `vadd`, how is it related to `call`?

Also, `Quantize()` is also a func `call` in the IR?

Reply:
Actually I dont know why we should check `isVectorized` since before my changes this code is here.

`Quantize()` is deleted from codes.